Formula/spatialite-gui.rb

```diff
@@ -1,10 +1,9 @@
 class SpatialiteGui < Formula
   desc "GUI tool supporting SpatiaLite"
   homepage "https://www.gaia-gis.it/fossil/spatialite_gui/index"
-  url "https://www.gaia-gis.it/gaia-sins/spatialite-gui-sources/spatialite_gui-1.7.1.tar.gz"
-  sha256 "cb9cb1ede7f83a5fc5f52c83437e556ab9cb54d6ace3c545d31b317fd36f05e4"
+  url "https://www.gaia-gis.it/gaia-sins/spatialite-gui-sources/spatialite_gui-2.1.0-beta1.tar.gz"
+  sha256 "ba48d96df18cebc3ff23f69797207ae1582cce62f4596b69bae300ca3c23db33"
```

Comment:
We could probably also do this version bump at the same time as we migrate `gdal`'s dep tree to `proj`.

Reply:
In that case, I think I should close this PR and do this in https://github.com/Homebrew/homebrew-core/pull/94807 instead.